integration/generator/resources/ec2_linux_test_matrix.json

```diff
@@ -2,14 +2,27 @@
   {
     "os": "ubuntu",
     "username": "ubuntu",
+    "instanceType":"t3a.xlarge",
     "installAgentCommand": "sudo dpkg -i -E ./amazon-cloudwatch-agent.deb",
     "ami": "cloudwatch-agent-integration-test-ubuntu*",
     "caCertPath": "/etc/ssl/certs/ca-certificates.crt",
-    "arc": "amd64", "binaryName": "amazon-cloudwatch-agent.deb"
+    "arc": "amd64",
+    "binaryName": "amazon-cloudwatch-agent.deb"
+  },
+  {
+    "os": "debian-10-arm64",
```

Comment:
Why do we have to specify `arm64` in the os name here if the `arc` value already tells us that?

Reply:
Mainly because of this reason: 
* The [cache key](https://github.com/aws/amazon-cloudwatch-agent/blob/master/.github/workflows/integrationTest.yml#L419) is using `os` instead of aarch ( we can change that). Therefore, if there is a similar os but in different aarch, it would return a similar key when one of the test executes successfully and return a false positive.

Reply:
Then doesn't it make more sense to update the cache key instead of this?

Reply:
Yes. I have changed the [cache key](https://github.com/aws/amazon-cloudwatch-agent/pull/500/files#diff-9b9f9a38b889c9dcb55ba0330baa69c8fdd62ac1100b6eb59b6fc44008f5d320R414) in the latest commit 👍